tueplots/fonts.py

```diff
@@ -33,6 +33,38 @@ def neurips2021_tex(*, family="serif"):
     }
 
 
+def neurips2022(*, family="serif"):
+    """Fonts for Neurips 2022."""
+    return {
+        "text.usetex": False,
+        "font.serif": ["Times New Roman"],
+        "mathtext.fontset": "stix",  # free ptmx replacement, for ICML and NeurIPS
+        "mathtext.rm": "Times New Roman",
+        "mathtext.it": "Times New Roman:italic",
+        "mathtext.bf": "Times New Roman:bold",
+        "font.family": family,
+    }
+
+
+def neurips2022_tex(*, family="serif"):
+    """Fonts for Neurips 2022. LaTeX version."""
+    preamble = r"\renewcommand{\rmdefault}{ptm}\renewcommand{\sfdefault}{phv}"
+    if family == "serif":
+        return {
+            "text.usetex": True,
+            "font.family": "serif",
+            "text.latex.preamble": preamble,
+        }
+    preamble += (
+        r"\renewcommand{\familydefault}{\sfdefault} \usepackage{sansmath} \sansmath"
+    )
+    return {
+        "text.usetex": True,
+        "font.family": "sans-serif",
+        "text.latex.preamble": preamble,
+    }
+
+
```

Comment:
This is identical to `neurips2021`, right? If so, how do you feel about extracting the content of this function into some `_neurips_base(...)` function, and let `neurips2021` and `neurips2022` point there? This way there should be a little less code duplication, and it might scale better to future editions. What do you think? 

(The fontsizes do this to some extent already, and the figsize appears to be different. So this only applies to the fonts I guess. Am I correct?)

Reply:
Good idea! I'm gonna take care of this...